src/Raven.Server/Documents/Handlers/Debugging/DocumentDebugHandler.cs

```diff
@@ -54,5 +57,89 @@ public async Task HugeDocuments()
                 writer.WriteEndObject();
             }
         }
+        
+        [RavenAction("/databases/*/debug/documents/scan-corrupted-ids", "GET", AuthorizationStatus.ValidUser, EndpointType.Read)]
+        public async Task ScanCorruptedIds()
```

Comment:
As we talked directly, please consider if it needs to account for inconsistencies between lower ID and actual ID after the recent change https://github.com/ravendb/ravendb/pull/20645